pkg/controllers/dynakube/oneagent/daemonset/daemonset_test.go

```diff
@@ -344,7 +344,7 @@ func TestHostMonitoring_SecurityContext(t *testing.T) {
 		dk := dynakube.DynaKube{
 			ObjectMeta: metav1.ObjectMeta{
 				Annotations: map[string]string{
-					exp.OAPrivilegedKey: "true",
+					exp.OAPrivilegedKey: annotationTrueValue,
```

Comment:
We discussed before that we want to use string literals in tests, so pls undo this change.